src/components/ApiRefRouting/ScrollRouter.js

```diff
@@ -36,16 +37,19 @@ export const ScrollRouter = ({ children, initialActive = "" }) => {
   React.useEffect(() => {
     const handler = throttle(() => {
       // If we haven't scrolled at least 100 pixels, just bail.
-      if (Math.abs(window.scrollTop - lastScrollPosition) < 100) {
+      if (Math.abs(window.scrollY - lastScrollPosition) < 100) {
         return;
       }
-      isScrollingDown.current = window.scrollTop > lastScrollPosition;
-      lastScrollPosition = window.scrollTop;
+      isScrollingDown.current = window.scrollY > lastScrollPosition;
+      lastScrollPosition = window.scrollY;
 
       const newActiveNode = findActiveNode(
         trackedElementsRef.current,
-        isScrollingDown.current,
+        isNavClicked ? true : isScrollingDown.current,
       );
+
+      if (isNavClicked) onNavClick(!isNavClicked);
```

Comment:
looks like this can just be `onNavClick(false)` since that's the end result you're looking for